tests/std/tests/GH_002620_construct_at_workaround/test.cpp

```diff
@@ -27,7 +27,7 @@ struct copy_elider {
 int main() {
     U u;
     // GH-2620: <xutility>: SFINAE constraint on construct_at prevents emplacing immovable objects with copy elision
-#ifndef __EDG__ // TRANSITION, DevCom-XXXX
+#ifndef __EDG__ // TRANSITION, DevCom-10000388

```

Comment:
With the `#ifdef __EDG__` this should always work now

Reply:
Maybe I wrote something wrong at the issue. EDG gives an error for both variants.
Look DevCom-10000388 :
```
$ cl main.cpp /nologo /std:c++latest /BE
main.cpp
"main.cpp", line 37: error: no instance of function template "construct_at1"
          matches the argument list
            argument types are: (S *, copy_elider)
      construct_at1(&u.s, copy_elider{});
      ^

"main.cpp", line 38: error: no instance of function template "construct_at2"
          matches the argument list
            argument types are: (S *, copy_elider)
      construct_at2(&u.s, copy_elider{});
      ^
```